lib/ecmascript.ts

```diff
@@ -3391,8 +3421,8 @@ export function UnbalanceDateDurationRelative(dateDuration: DateDuration, plainR
   // balance years, months, and weeks down to days
   const isoDate = TemporalObjectToISODateRecord(plainRelativeTo);
   const later = CalendarDateAdd(GetSlot(plainRelativeTo, CALENDAR), isoDate, yearsMonthsWeeksDuration, 'constrain');
-  const epochDaysEarlier = ISODateToEpochDays(isoDate.year, isoDate.month, isoDate.day);
-  const epochDaysLater = ISODateToEpochDays(later.year, later.month, later.day);
+  const epochDaysEarlier = ISODateToEpochDays(isoDate.year, isoDate.month - 1, isoDate.day);
```

Comment:
I don't know why these -1s are here, so I'll assume they're correct!

Reply:
Just to match the spec text: https://tc39.es/proposal-temporal/#sec-isodatetoepochdays